listenbrainz/webserver/templates/index/add-data.html

```diff
@@ -80,6 +80,9 @@ <h3>Playlist submissions and tools</h3>
   <h4>Tools</h4>
   <ul>
     <li><em><a href="https://github.com/Serene-Arc/listenbrainz-playlist-uploader">listenbrainz-playlist-uploader</a></em>, a CLI tool for submitting local M3U playlists to ListenBrainz, as well as submitting feedback on tracks</li>
+    <li><em><a href="https://github.com/regorxxx/ListenBrainz-SMP">ListenBrainz-SMP</a></em>, a <em><a href="https://www.foobar2000.org/">Foobar2000</a></em> plugin for submitting and retrieving playlists from ListenBrainz (+ Spotify), as well as retrieving recommendations or submitting feedback on tracks.
+    <li><em><a href="https://github.com/regorxxx/Playlist-Manager-SMP">Playlist-Manager-SMP</a></em>, a <em><a href="https://www.foobar2000.org/">Foobar2000</a></em> plugin for syncing local playlists (in multiple formats) with ListenBrainz (+ Spotify). Tracks playlists changes and also allows to resolve tracks with local content and YouTube links.
+    <li><em><a href="https://github.com/regorxxx/Wrapped-S">Wrapped-SMP</a></em>, a <em><a href="https://www.foobar2000.org/">Foobar2000</a></em> plugin for creating reports based on user listens similar to the one found at Spotify. Suggested playlists use ListenBrainz recommendations (without requiring listens upload to the server).
```

Comment:
Looks like the link got borked during copy/paste:
```suggestion
    <li><em><a href="https://github.com/regorxxx/Wrapped-SMP">Wrapped-SMP</a></em>, a <em><a href="https://www.foobar2000.org/">Foobar2000</a></em> plugin for creating reports based on user listens similar to the one found at Spotify. Suggested playlists use ListenBrainz recommendations (without requiring listens upload to the server).
```